build/media-suite_helper.sh

```diff
@@ -29,14 +29,23 @@ do_print_status() {
     local status="$3"
     local pad
     pad=$(printf '%0.1s' "."{1..72})
-    local padlen=$((ncols-${#name}-${#status}-3))
-    printf '%s%*.*s [%s]\n' "${bold}$name${reset}" 0 \
-        "$padlen" "$pad" "${color}${status}${reset}"
+    if [[ $logTime = y ]]; then
+        printf '%(%H:%M:%S)T %s%*.*s [%s]\n' -1 "${bold}$name${reset}" 0 \
+        "$((ncols-${#name}-${#status}-12))" "$pad" "${color}${status}${reset}"
+    else
+        printf '%s%*.*s [%s]\n' "${bold}$name${reset}" 0 \
+        "$((ncols-${#name}-${#status}-3))" "$pad" "${color}${status}${reset}"
+    fi
 }
 
 do_print_progress() {
     if [[ $logging = y ]]; then
-        [[ ${1} =~ ^[a-zA-Z] ]] && echo "├ $*..." || echo -e "$*..."
+        if [[ $logTime = y ]]; then
+            [[ ${1} =~ ^[a-zA-Z] ]] && printf '%(%H:%M:%S)T %s\n' -1 \
+            "${bold}├${reset} $*..." \|| printf '%(%H:%M:%S)T %s\n' -1 "$*..."
+        else
+            [[ ${1} =~ ^[a-zA-Z] ]] && echo "${bold}├${reset} $*..." || echo -e "$*..."
+        fi
     else
         set_title "$* in $(get_first_subdir)"
         echo -e "${bold}$* in $(get_first_subdir)${reset}"
```

Comment:
Do you suppose the same should be done for the non-logging option?

Reply:
Only for the `Runnin * in *` and `* svn ... [Updated]`

Reply:
Sure.